### PR TITLE
Add test infrastructure for index.js of Disputer bot

### LINKED ISSUE #1113: Add a way to test index.js for liquidator and dispute bots
To make sure the logic in those files works as expected (both have nontrivial logic), we should have some way to test them -- possibly a dry run mode that causes them to break out of their iteration after one loop or something.

### PR DESCRIPTION
**Pull request by nicholaspai**
Resolves #1113 

Same structure as #1116 but for Disputer bot

### CODE REVIEW

File: disputer/test/Disputer.js [hunk 1]
```diff
@@ -30,15 +30,6 @@ contract("Disputer.js", function(accounts) {
   let syntheticToken;
   let mockOracle;
 
-  // States for Liquidation to be in
-  const STATES = {
```
Comment (nicholaspai) [inline disputer/test/Disputer.js]: Unused variable

File: disputer/index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const argv = require("minimist")(process.argv.slice(), { string: ["address", "price"] });
-const { toWei, hexToUtf8, toBN } = web3.utils;
```
Comment (nicholaspai) [inline disputer/index.js]: Unused variable

File: liquidator/index.js [hunk 3]
```diff
@@ -75,7 +75,7 @@ const Poll = async function(callback) {
       throw new Error("Bad input arg! Specify a `price` as the pricefeed.");
     }
 
-    await run(argv.address, argv.price, true);
+    await run(argv.price, argv.address, true);
```
Comment (nicholaspai) [inline liquidator/index.js]: I realize in I had reversed input of these variables

File: disputer/test/index.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+    // Deploy a new expiring multi party
+    emp = await ExpiringMultiParty.new(constructorParams);
+
+    // Create a new instance of the ExpiringMultiPartyClient & GasEstimator to construct the disputer
+    empClient = new ExpiringMultiPartyClient(ExpiringMultiParty.abi, web3, emp.address);
+    gasEstimator = new GasEstimator();
+
+    // Create a new instance of the disputer to test
+    disputer = new Disputer(empClient, gasEstimator, disputeBot);
```
Comment (mrice32) [inline disputer/test/index.js]: Do we need any of this code? Doesn't the dispute bot initialize all of this itself?

File: disputer/test/index.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  before(async function() {
+    collateralToken = await Token.new({ from: contractCreator });
+    await collateralToken.addMember(1, contractCreator, {
+      from: contractCreator
+    });
+
+    // Create a mockOracle and finder. Register the mockMoracle with the finder.
+    mockOracle = await MockOracle.new(IdentifierWhitelist.address, {
+      from: contractCreator
+    });
```
Comment (mrice32) [inline disputer/test/index.js]: Technically I think we can do without much of this setup as well, but I understand if you think we'll be adding more tests here that depend on it.

File: disputer/test/index.js (hunk 4 above)
Comment (nicholaspai) [reply disputer/test/index.js]: No, it looks like you pass these into the Disputer just like the Liquidator..perhaps I'm misunderstanding your question?

File: disputer/test/index.js (hunk 5 above)
Comment (nicholaspai) [reply disputer/test/index.js]: True, I am removing mock oracle logic and `addMember`, will do so from liquidator bot test as well

Comment (nicholaspai) [reply disputer/test/index.js]: done

File: disputer/test/index.js (hunk 4 above)
Comment (mrice32) [reply disputer/test/index.js]: Hmmmm, I don’t see `disputer` declared above or used in the test. Am I missing something or do we not have to worry about creating the disputer here?

Comment (nicholaspai) [reply disputer/test/index.js]: oh duh you're right.

Comment (nicholaspai) [reply disputer/test/index.js]: Will make the change for liquidator as well.

**Pull request merged**